raiden/network/transport/udp/udp_transport.py

```diff
@@ -168,14 +168,15 @@ def single_queue_send(
 class UDPTransport(Runnable):
     UDP_MAX_MESSAGE_SIZE = 1200
 
-    def __init__(self, discovery, udpsocket, throttle_policy, config):
+    def __init__(self, address, discovery, udpsocket, throttle_policy, config):
```

Comment:
Why are you introducing the address to the transport layer here? I can't see it from looking at the PR as is. I only see it used in the logs [here](https://github.com/raiden-network/raiden/pull/2309/files#diff-57baa341c1b2c4e11a40560e03e5b309L45). Any other reason?

Reply:
it fixes a race condition, it is explained here:

https://github.com/raiden-network/raiden/pull/2309/files#diff-72b8041548bf416a299ac073c02e24e5R442

Reply:
this is not required anymore after #2472